fossunited/fossunited/web_template/home_page___events/home_page___events.html

```diff
@@ -0,0 +1,374 @@
+        border: none;
+        padding: 0.8rem 1.5rem;
+        color: var(--green-800, #16794C);
+        font-size: 1rem;
+        font-style: normal;
+        font-weight: 400;
+        line-height: 115%; /* 1.15rem */
+        letter-spacing: -0.02rem;
+    }
+
+    .mobile-only{
+        display: none;
+    }
+
+    @media screen and (max-width: 768px) {
+
+        .events-section{
+            padding: 0rem 2rem 0rem 2rem;
+
+        }
+
+
+        .events-container{
+            max-width: 21.9375rem;
+            gap: 7rem;
+        }
+
+        .events-block{
+            grid-template-columns: 1fr;
+            gap: 2.5rem;
+            width: 100%
+        }
+
+        .flagship-event-card{
+            border-radius: 0.5rem;
+            border: 0.5px solid var(--Gray-200, #EAECF0)
+                }
+
+        .view-events-link{
+            display: none;
+        }
+
+        .mobile-only{
+            display: block;
+        }
+
+    }
+
+
+
+</style>
+
+
+
+<div class="events-section">
+    <div class="events-container">
+
+        <!-- Flagship Events Section -->
+        <div class="events-subsection">
+            <h2>Flagship Events</h2>
+
+            <div class="events-block">
+
+                <a class="flagship-event-card" href="">
+                    <img class="flagship-card-image" src="assets/fossunited/icons/indiafoss-icon.svg" alt="IndiaFOSS">
+                    <h3 class="flagship-card-heading">IndiaFOSS</h3>
+                    <p class="flagship-card-desc">IndiaFOSS is an annual Free and Open Source Software (FOSS) conference,
+                        organised with passion and commitment by the FOSS United community. </p>
+
+                    <p class="flagship-card-cta">More >></p>
+
+                </a>
+
+                <a class="flagship-event-card" href="">
+
+                    <img class="flagship-card-image" src="assets/fossunited/icons/fosshack-icon.svg" alt="IndiaFOSS">
+                    <h3 class="flagship-card-heading">FOSS Hack</h3>
+                    <p class="flagship-card-desc">FOSS Hack is our community-run flagship hackathon to promote Free and Open Source Software by bringing
+                        together students and professionals to build or extend FOSS projects.</p>
+
+                    <p class="flagship-card-cta">More >></p>
+
+                </a>
+
+            </div>
+        </div>
+
+        <!-- Upcoming Events Section -->
+
+        <div class="events-subsection">
+
+            <div class="upcoming-events-header">
+                <h2>Upcoming Events</h2>
+                <a class="view-events-link" href="{{events_page_link}}">VIEW ALL EVENTS ↗</a>
+        </div>
+
+        <div class="events-block">
+            {% set events = frappe.get_all("FOSS Chapter Events", fields=["event_name", "must_attend", "event_location", "map_link", "event_start_date"], filters={"status": "Approved"}, page_length=3, order_by='event_start_date') %}
```

Comment:
Will page_length 3 will get only 3 entries? If so, that is too low in my opinion. We should atleast show 6 if the grid is a 3 column

Reply:
Please clarify @RiyaMathew-11